docs/rule/kotlin_library.soy

```diff
@@ -0,0 +1,188 @@
+{namespace kotlin_library}
```

Comment:
If most of the arguments are the same as `java_library`, it might make sense to DRY this up and use a template that is commonly used for other jvm based rule docs